x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_exceptions/components/flyout_components/item_conditions/index.tsx

```diff
@@ -212,7 +212,11 @@ const ExceptionsConditionsComponent: React.FC<ExceptionsFlyoutConditionsComponen
       </SectionHeader>
       {includesRuleWithEQLSequenceStatement && (
         <>
-          <EuiCallOut data-test-subj="eqlSequenceCallout" title={eqlCalloutWarning} />
+          <EuiCallOut
+            announceOnMount={false}
```

Comment:
```suggestion
            announceOnMount
```

Should this one be true?

It's conditionally rendered